src/csvcubed/writers/qbwriter.py

```diff
@@ -89,10 +91,20 @@ def write(self, output_folder: Path):
         # Map all labels to their corresponding URI-safe-values, where possible.
         # Also converts all appropriate columns to the pandas categorical format.
 
-        _logger.info("Beginning CSV-W Generation")
-
+        _logger.info(f"Beginning CSV-W Generation: {self.csv_file_name}")
+        
         ensure_int_columns_are_ints(self.cube)
 
+        # Bring the pandas representation of bolleans inline with what the csvw spec requires
+        # True != true, False != false
+        assert isinstance(self.cube.data, pd.DataFrame)
```

Comment:
Can you change this to an if statement so if there isn't any data it just doesn't try to do this instead of bailing out with an extension?